### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by jugarrit**
@jpjoyal @lemieux @chloepouprom @dannytranlx @alavers

### CODE REVIEW

File: src/js/components/channels/twilio-channel-content.jsx [hunk 1]
```diff
@@ -71,21 +71,29 @@ export class TwilioChannelContentComponent extends Component {
                                                         onClick={ this.linkTwilioNumber }>
                                                     Continue
                                                 </button> : '';
+
+        const onEnterKeyPress = appUserNumberValid ? this.linkTwilioNumber : () => {
```
Comment (lemieux) [inline src/js/components/channels/twilio-channel-content.jsx]: that's some weird formatting :(

File: src/js/lib/react-telephone-input.jsx [hunk 2]
```diff
@@ -57,7 +57,8 @@ export class ReactTelephoneInput extends Component {
         autoFormat: true,
         onlyCountries: allCountries,
         defaultCountry: allCountries[0].iso2,
-        onEnterKeyPress() {},
+        onEnterKeyPress() {
```
Comment (lemieux) [inline src/js/lib/react-telephone-input.jsx]: whaa?

**Comment by lemieux** [comment]
lgtm :+1:

File: README.md [hunk 3]
```diff
@@ -148,6 +148,7 @@ var skPromise = Smooch.init({
         messengerChannelDescription: 'Connect your Facebook Messenger account to be notified when you get a reply and carry the conversation on Facebook Messenger.',
         frontendEmailChannelDescription: 'To talk to us using email just send a message to our email address and we\'ll reply shortly:',
         smsChannelDescription: 'To talk to us using SMS, just send a text message to this number from your favorite SMS app:',
+        smsChannelPendingDescription: 'Check your messages at ${number} to confirm your phone number.',
```
Comment (mspensieri) [inline README.md]: the convention so far has been just `{}` for variables, not `${}`: https://github.com/smooch/smooch-js/pull/340/files#diff-04c6e90faac2675aa89e2176d2eec7d8L141

File: src/js/components/channels/twilio-channel-content.jsx (hunk 1 above)
Comment (jugarrit) [inline src/js/components/channels/twilio-channel-content.jsx]: Might be from my setup

File: src/js/lib/react-telephone-input.jsx (hunk 2 above)
Comment (jugarrit) [inline src/js/lib/react-telephone-input.jsx]: I'll re-save this page to fix it :P

File: README.md (hunk 3 above)
Comment (jugarrit) [inline README.md]: That was a mistake! Will fix

File: src/js/components/channels/twilio-channel-content.jsx (hunk 1 above)
Comment (jugarrit) [inline src/js/components/channels/twilio-channel-content.jsx]: Hmm actually it's happening here too

Comment (jugarrit) [inline src/js/components/channels/twilio-channel-content.jsx]: I could create a noop function to fix it maybe?

**Comment by chloepouprom** [comment]
👍

**Pull request merged**